rise/lib/services/questions.ts

```diff
@@ -12,3 +13,36 @@ export const getQuestions: QueryResolvers['questions'] = async () => {
   `
   )
 }
+
+export async function latestAnswerByUser(id: string): Promise<any[]> {
```

Comment:
Same here, (`Answer` is from the generated Graphql-types):
```suggestion
export async function latestAnswerByUser(id: string): Promise<Answer | null> {
```